build/compile_descriptor

```diff
@@ -20,20 +20,26 @@ is_up_to_date () {
 
 set -e
 
+if [ -d "./ebin" ]; then
+    OUT="./ebin"
+else
+    OUT="$REBAR_DEPS_DIR/gpb/ebin"
+fi
+
```

Comment:
Ah, you mean running `rebar3` after `rebar`. That is actually not supported by rebar3, we tell people and document to always blow away `./ebin` and `deps` (unless you know better for whatever reason).